expyfun/stimuli/_tracker.py

```diff
@@ -0,0 +1,858 @@
+        The size the step when the tracker moves up. If float it will stay the
+        same. If list of float then it will change when when
+        ``change_criteria`` are encountered. See note below for more specific
+        information on dynamic tracker parameters specified with a list.
+    step_size_down : float | list of float
+        The size the step when the tracker moves down. If float it will stay
+        the same. If list of float then it will change when when
+        ``change_criteria`` are encountered. See note below for more specific
+        information on dynamic tracker parameters specified with a list.
+    stop_criterion : int
+        The number of reversals or trials that will stop the tracker.
+    stop_rule : str
+        How to determined when the tracker stops. Can be 'reversals' or
+        'trials.'
+    start_value : float
+        The starting level of the tracker.
+    change_criteria : list of int | None
+        The points along the tracker to change its step sizes. Has an effect
+        where ``step_size_up`` and ``step_size_down`` are
+        lists. The length of ``change_criteria`` must be the same as the
+        length of ``step_size_up`` and ``step_size_down``. See note below for
+        more specific usage information. Should be None if static step sizes
+        are used.
+    change_rule : str
+        Whether to change paramters based on 'trials' or 'reversals'.
+    x_min : float
+        The minimum value that the tracker level (``x``) is allowed to take.
+    x_max : float
+        The maximum value that the tracker level (``x``) is allowed to take.
+
+    Returns
+    -------
+    tracker : instance of TrackerUD
+        The up-down tracker object.
+
+    Notes
+    -----
+    It is common to use dynamic parameters in an adaptive tracker. For example:
+    the step size is often large for the first couple reversals to get in the
+    right general area, and then it is reduced for the remainder of the track.
+    This class allows that functionality by defining that parameter with a list
+    of values rather than a scalar. The parameter will change to the next value
+    in the list whenever a change criterion (number of trials or reversals) is
+    encountered. This means that the length of the list defining a dynamic
+    parameter must always be the same as that of ``change_criteria``, and the
+    first element of change_criteria must always be 0.
+    For the example given above:
+        ``step_size_up=[1., 0.2]``, ``step_size_down=[1., 0.2]``,
+        ``change_criteria=[0, 2]``, ``change_rule='reversals'``
+    would change the step sizes from 1 to 0.2 after two reversals.
+
+    If dynamic step sizes are used, both ``step_size_up`` and
+    ``step_size_down`` must have the same length as each other and
+    ``change_cirteria``. If static step sizes are used, both ``step_size_up``
+    and ``step_size_up`` must be scalars and ``change_criteria`` must be None.
```

Comment:
first sentence of this paragraph is redundant with the final sentence of previous paragraph (and also has a misspelled "change_cirteria").  I recommend deleting first sentence of this one.